internal/controller/replication.storage/status.go

```diff
@@ -48,24 +51,28 @@ func setPromotedCondition(conditions *[]metav1.Condition, observedGeneration int
 // sets conditions when volume promotion was failed.
 func setFailedPromotionCondition(conditions *[]metav1.Condition, observedGeneration int64) {
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume failed to promote",
```

Comment:
Make error/failed cases a little more clear by starting the message with _failed_ or _error_.